language-server/modules/langserver-core/src/test/resources/codeaction/type-guard/source/typeGuard2.bal

```diff
@@ -0,0 +1,10 @@
+type Token SingleCharDelim|MultiCharDelim|Keyword;
+type SingleCharDelim "+" | "-" ;
+type MultiCharDelim "<<" | ">>" | CompoundAssignDelim;
+type CompoundAssignDelim "+=" | "-=" ;
+type Keyword KEY | "string";
```

Comment:
Let's change this to `type Keyword KEY | "string" | string | 100;`

Reply:
Current fix doesn't work for this example (adds quotation marks around 100 as well). Therefore, blocked until #32569 is fixed.